androbot/handlers.py

```diff
@@ -168,14 +168,15 @@ async def after_select_answer_type(message: aiotypes.Message, state: FSMContext)
 
 
 @dp.message_handler(regexp="Отмена", state=DialogueStates.ARE_YOU_READY_FOR_TEST)
-@dp.message_handler(regexp="Главное меню", state=DialogueStates.GOT_ANSWER)
+@dp.message_handler(regexp="Главное меню", state=DialogueStates.NO_ANSWER)
+@dp.message_handler(regexp="Главное меню", state=DialogueStates.ANSWER_SCORED_BY_USER)
 @dp.message_handler(regexp="Главное меню", state=DialogueStates.HAS_STARTED_TEST)
 @dp.message_handler(regexp="Главное меню", state=DialogueStates.USER_SCORE)
```

Comment:
```suggestion
@dp.message_handler(regexp="Отмена|Главное меню", state=DialogueStates.ARE_YOU_READY_FOR_TEST | DialogueStates.NO_ANSWER | DialogueStates.ANSWER_SCORED_BY_USER | DialogueStates.HAS_STARTED_TEST | DialogueStates.USER_SCORE)
```
Есть причины не переделать так везде?

Reply:
а `|` это легитимный оператор?
что-то в консоли такое не проходит. попробую сделать

Reply:
Я видел в доке, что так делают..